translations/tiled_de.ts

```diff
@@ -593,17 +593,17 @@
     <message>
         <location line="+8"/>
         <source>&amp;Never</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nie</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>For &amp;Selected Objects</source>
-        <translation type="unfinished"></translation>
+        <translation>Für ausge&amp;wählte Objekte</translation>
```

Comment:
Hm. This changes the accessor key... @bjorn @wayfu wouldn't it make sense to put the `&` before the `s` in the German part too?